<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scrapy 数据导出工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1061</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>过滤</string>
    </property>
    <widget class="QCheckBox" name="check_box_filter_image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤图片下载失败的产品</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_filter_repeat_product">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>sku</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_filter_min_price">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤最低价格</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_filter_invalidate_product">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤无效产品</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤重复产品</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_check_data">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>检查</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>1061</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>数据</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>总记录数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_total_quantity">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>筛选后记录数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_validate_quantity">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_image_failed_quantity">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>图片下载失败数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>重复数据数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_repeat_quantity">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_export_data">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>导出</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>1061</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>日志</string>
    </property>
    <widget class="QTextEdit" name="textarea_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1041</width>
       <height>181</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1061</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>加载</string>
    </property>
    <widget class="QPushButton" name="button_load_project">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>加载项目文件(excel)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_project_file">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>21</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="button_set_default_brand">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>设置默认品牌</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_default_brand">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1061</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>尺码修正</string>
    </property>
    <widget class="QComboBox" name="combo_box_category">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>待替换的类别</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_update_size">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>70</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_size">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>491</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>尺码模板</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combobox_size_temolate">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>281</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_category_pattern">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>601</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>1061</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>产品类别修正</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>待替换的类别</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combobox_category_name">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_category_name_pattern">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>601</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_new_category_name">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>821</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_upate_category_name">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>70</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>70</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>新类别</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
